[2022-10-27 02:20:19,821] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2022-10-27T02:20:17.398602+00:00 [queued]>
[2022-10-27 02:20:19,863] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2022-10-27T02:20:17.398602+00:00 [queued]>
[2022-10-27 02:20:19,866] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-27 02:20:19,868] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-27 02:20:19,869] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-27 02:20:19,976] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_table> on 2022-10-27 02:20:17.398602+00:00
[2022-10-27 02:20:20,121] {standard_task_runner.py:52} INFO - Started process 348 to run task
[2022-10-27 02:20:20,182] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2022-10-27T02:20:17.398602+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp33hwo9v5', '--error-file', '/tmp/tmp551uwr1z']
[2022-10-27 02:20:20,194] {standard_task_runner.py:80} INFO - Job 13: Subtask create_table
[2022-10-27 02:20:20,371] {task_command.py:369} INFO - Running <TaskInstance: user_processing.create_table manual__2022-10-27T02:20:17.398602+00:00 [running]> on host 18112c9f481e
[2022-10-27 02:20:20,570] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-10-27T02:20:17.398602+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-27T02:20:17.398602+00:00
[2022-10-27 02:20:20,589] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-10-27 02:20:20,597] {dbapi.py:208} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS users (
            firstname TEXT NOT NULL,
            lastname TEXT NOT NULL,
            country TEXT NOT NULL,
            username TEXT NOT NULL,
            password TEXT NOT NULL,
            email TEXT NOT NULL
        );
        , parameters: None
[2022-10-27 02:20:20,613] {postgres.py:94} INFO - NOTICE:  relation "users" already exists, skipping

[2022-10-27 02:20:20,651] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20221027T022017, start_date=20221027T022019, end_date=20221027T022020
[2022-10-27 02:20:20,717] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-27 02:20:20,770] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
